core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-ema-info-display/dot-ema-info-display.component.ts

```diff
@@ -0,0 +1,127 @@
+import { NgIf } from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Input,
+    OnChanges,
+    inject,
+    signal
+} from '@angular/core';
+import { Router } from '@angular/router';
+
+import { ButtonModule } from 'primeng/button';
+
+import { DotExperiment } from '@dotcms/dotcms-models';
+import { DotMessagePipe } from '@dotcms/ui';
+
+import { EditEmaStore } from '../../../dot-ema-shell/store/dot-ema.store';
+import { EDITOR_MODE } from '../../../shared/enums';
+import { EditorData } from '../../../shared/models';
+import { getIsDefaultVariant } from '../../../utils';
+
+interface InfoOptions {
+    icon: string;
+    info: {
+        message: string;
+        args: string[];
+    };
+    action?: () => void;
+    actionIcon?: string;
+}
+
+@Component({
+    selector: 'dot-ema-info-display',
+    standalone: true,
+    imports: [NgIf, ButtonModule, DotMessagePipe],
+    templateUrl: './dot-ema-info-display.component.html',
+    styleUrls: ['./dot-ema-info-display.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotEmaInfoDisplayComponent implements OnChanges {
+    @Input() editorData: EditorData;
```

Comment:
no `editorData: input<EditorData>` ?

Reply:
did jest fix the issue?

Reply:
Yes, I updated that a few weeks ago, go with it 🤛🏼.
https://github.com/thymikee/jest-preset-angular/pull/2303.  `input` signals, `outputs`, `query`, and `model`.

Reply:
Alright! I'll test this feature in another moment. Right now I need this merged

Reply:
sure. Dont worry.